spec/models/miq_action_spec.rb

```diff
@@ -159,4 +159,44 @@
       end
     end
   end
+
+  context '.create_default_actions' do
+    context 'miq_actions table is initially empty' do
+      before { MiqAction.destroy_all }
+
+      context 'seeding default actions from csv' do
+        before { MiqAction.create_default_actions }
+
+        it 'should create 34 new actions' do
+          expect(MiqAction.count).to eq 34
```

Comment:
The tests are brittle when tied to the count like this.  I was thinking of stubbing the `fname.read` call and return a small sub-set of actions to test against.  Would make the tests a little quicker as well.  Thoughts?